Update 08-configs.md (#19377)
docker/docs: content/compose/compose-file/08-configs.md

The following
example modifies the previous one to look up a config using the parameter `HTTP_CONFIG_KEY`. The

> actual lookup key will be set at deployment time by the [interpolation](12-interpolation.md) of

Author: ```suggestion
actual lookup key is set at deployment time by the [interpolation](12-interpolation.md) of
```
